src/main/resources/assets/securitycraft/lang/de_de.json

```diff
@@ -461,11 +461,11 @@
     "gui.securitycraft:scManual.recipe.keypad": "Platziere einen Rahmen und rechtsklicke ihn mit einem Nummernfeld.",
     "gui.securitycraft:scManual.recipe.keypad_chest": "Platziere eine Truhe und rechtsklicke sie mit einem Nummernfeld.",
     "gui.securitycraft:scManual.recipe.keypad_furnace": "Platziere einen Ofen und rechtsklicke ihn mit einem Nummernfeld.",
-    "gui.securitycraft:scManual.recipe.reinforced": "Jeder verstärkte Block kann wie folgt gecraftet werden, wenn nicht anders in diesem Buch erwähnt: Platziere einen normalen Block (z.B. Verstärkter Stein -> Stein) und baue ihn mit einem Universellen Block Verstärker ab, oder rechtsklicke den Universellen Blockverstärker, lege ihn in den Slot und schließe das GUI.",
-    "gui.securitycraft:scManual.recipe.reinforced_iron_trapdoor": "Jeder verstärkte Block kann wie folgt gecraftet werden, wenn nicht anders in diesem Buch erwähnt. Platziere einen normalen Block (z.B. Verstärkter Stein -> Stein) und zerbreche ihn mit einem Universellen Block Verstärker, oder rechtsklicke den Universellen Blockverstärker, lege ihn in den Slot und schließe das GUI.",
+    "gui.securitycraft:scManual.recipe.reinforced": "Jeder verstärkte Block kann wie folgt hergestellt werden, wenn nicht anders in diesem Buch erwähnt: Platziere einen normalen Block (z.B. Verstärkter Stein -> Stein) und baue ihn mit einem Universellen Block Verstärker ab, oder rechtsklicke den Universellen Blockverstärker, lege ihn in den Slot und schließe das GUI.",
+    "gui.securitycraft:scManual.recipe.reinforced_iron_trapdoor": "Jeder verstärkte Block kann wie folgt hergestellt werden, wenn nicht anders in diesem Buch erwähnt. Platziere einen normalen Block (z.B. Verstärkter Stein -> Stein) und zerbreche ihn mit einem Universellen Block Verstärker, oder rechtsklicke den Universellen Blockverstärker, lege ihn in den Slot und schließe das GUI.",
     "gui.securitycraft:scManual.recipe.adminTool": "Das Admin-Tool ist nur durch das Kreativinventar erhältlich, falls es in der Konfigurationsdatei aktiviert ist.",
     "gui.securitycraft:scManual.recipe.admin_tool": "Das Admin-Tool ist nur durch das Kreativinventar erhältlich, falls es in der Konfigurationsdatei aktiviert ist.",
-    "gui.securitycraft:scManual.recipe.sc_manual": "Die SecurityCraft Anleitung ist durch das Kreativinventar oder via \"/sc help\" (ohne Anführungszeichen) erhältlich. Alternativ kann man ein Buch mit einem Eisengitter zusammencraften, um die Anleitung zu bekommen.",
+    "gui.securitycraft:scManual.recipe.sc_manual": "Die SecurityCraft Anleitung ist durch das Kreativinventar oder via \"/sc help\" (ohne Anführungszeichen) erhältlich. Alternativ kann man ein Buch mit einem Eisengitter in eine Werkbank legen, um die Anleitung zu bekommen.",
```

Comment:
Good change, I like this! Couldn't think of a good way to formulate this previously.